README.md

```diff
@@ -4,22 +4,23 @@
 
 For Healthcare IT teams and solution architects who want to integrate clinical data currently in different formats, the FHIR® Converter is an open source project that enables the conversion of legacy formatted health data to FHIR, expanding the use cases for health data and enabling interoperability.  
 
-Right now, the FHIR Converter transforms HL7 v2 messages into FHIR bundles using templates that define the mappings between the two data formats. Leveraging the FHIR Converter organizations can customize or create their own mapping templates based on their HL7 v2 implementation and transform them into FHIR bundles. These FHIR bundles are returned for further data manipulation or can be immediately persisted into a FHIR server, such as the [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/). The FHIR Converter released to open source on Thursday March 6th, 2020.
+The FHIR Converter transforms HL7 v2 messages and C-CDA documents into FHIR bundles using templates that define the mappings between the two data formats. Leveraging the FHIR Converter, organizations can customize or create their own mapping templates based on their HL7 v2 or C-CDA implementation and transform them into FHIR bundles. These FHIR bundles are returned for further data manipulation or can be immediately persisted into a FHIR server, such as the [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/). The FHIR Converter released to open source on Thursday March 6th, 2020 with support for HL7 v2 to FHIR conversion. On Friday June 12th, 2020, C-CDA to FHIR conversion was added to the OSS FHIR Converter.
 
 The open-source FHIR Converter consists of the following functionality:
 
-1. A set of starting templates, leveraging [handlebars](https://handlebarsjs.com/), to translate HL7 v2 messages into FHIR bundles. These templates were generated based on the mappings defined by the [HL7 community](https://confluence.hl7.org/display/OO/2-To-FHIR+Project). As more mappings are defined, we will continue to release updated versions of these templates. Current examples are ADT-A01, OML_O21, ORU-R01, and VXU_V04.
+1. A set of starting templates, leveraging [handlebars](https://handlebarsjs.com/), to translate HL7 v2 messages or C-CDA documents into FHIR bundles. 
 1. A set of sample messages to accompany the released templates.

```

Comment:
Since C-CDA doesn't call their files 'messages' we have been using 'data' in the converter.

Reply:
Fixed throughout but double checking I got everywhere. I say 'data' when it's general, but otherwise CDA documents or HL7v2 message